frontend/src/app/Settings/Facility/Components/FacilityInformation.tsx

```diff
@@ -44,119 +54,162 @@ const FacilityInformation: React.FC<Props> = ({
+          value={formCurrentValues.facility?.streetTwo ?? undefined}
+          registrationProps={register("facility.streetTwo")}
+        />
+        <TextInput
+          label="City"
+          name="facility-city"
+          value={formCurrentValues.facility?.city ?? undefined}
+          registrationProps={register("facility.city")}
+        />
+        <TextInput
+          label="ZIP code"
+          name="facility-zipCode"
+          value={formCurrentValues.facility?.zipCode}
+          required
+          validationStatus={
+            errors?.facility?.zipCode?.type ? "error" : undefined
+          }
+          errorMessage={errors?.facility?.zipCode?.message}
+          registrationProps={register("facility.zipCode", {
+            required: "Facility ZIP code is required",
+            pattern: {
+              value: zipCodeRegex,
+              message: "Facility ZIP code is invalid",
+            },
+          })}
+          className="usa-input--medium"
+        />
+        <Dropdown
+          label="State"
+          name="facility-state"
+          selectedValue={formCurrentValues.facility?.state}
+          options={stateCodes.map((c) => ({ label: c, value: c }))}
+          defaultSelect
+          required
+          hintText={
+            fieldState?.error && (
+              <span className="display-block margin-top-05 usa-error-message">
+                See a{" "}
+                <a href={urls.FACILITY_INFO}>
+                  {" "}
+                  list of states where SimpleReport is supported
+                </a>
+                .
+              </span>
+            )
+          }
+          validationStatus={errors?.facility?.state?.type ? "error" : undefined}
+          errorMessage={errors?.facility?.state?.message}
+          selectClassName="usa-input--medium"
+          data-testid="facility-state-dropdown"
+          registrationProps={register("facility.state", {
+            required: "Facility state is required",
+            validate: {
+              liveJurisdiction: (state: string) =>
+                isLiveState(state) ||
+                `SimpleReport isn’t currently supported in ${getStateNameFromCode(
+                  state
+                )}.`,
+            },
+          })}
+        />
+        <TextInput
+          label="CLIA number"
+          hintText={
+            <a
+              href="https://www.cdc.gov/clia/LabSearch.html#"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Find my CLIA
+            </a>
+          }
+          name="cliaNumber"
+          value={formCurrentValues.facility?.cliaNumber}
+          required
+          validationStatus={
+            errors?.facility?.cliaNumber?.type ? "error" : undefined
+          }
+          errorMessage={errors?.facility?.cliaNumber?.message}
+          registrationProps={register("facility.cliaNumber", {
+            required: "Facility CLIA number is required",
```

Comment:
Do we need to gate the boolean here based on whether CO is selected in the dropdown? I can't seem to submit a blank CLIA on the deployed instance when CO is selected. Lmk if I'm misunderstanding

https://github.com/CDCgov/prime-simplereport/assets/29645040/9d10dfcc-fce1-48c2-908c-70d7d79e38b6

Reply:
[Based on what was implemented previously](https://github.com/CDCgov/prime-simplereport/blob/04597c50f73cdd72a78442af636469ed98a0418f/frontend/src/app/Settings/Facility/facilitySchema.ts#L96), no matter what state we want require the CLIA number but for CO we don't validate it.